ldaptor/test/test_ldiftree.py

```diff
@@ -572,9 +570,7 @@ def _cb_test_subtree_many_cb(self, r, got):
             self.bar,
             self.foo,
             ]
-        got.sort()
-        want.sort()
-        self.assertEqual(got, want)
+        six.assertCountEqual(self, expected, got)
```

Comment:
> In test test_subtree_many_cb(), I noticed the original collected the results, sorted, and compared. Your test just counts to make sure the same number of components are in the result. This isn't exactly the same test. Does another test check to make sure that the components that are returned are the correct components?

I think that `assertCountEqual` does the same thing.
It has a stupid name, but see docs https://docs.python.org/3.2/library/unittest.html#unittest.TestCase.assertCountEqual

Reply:
Yes-- you are correct.  I like the Py2 name better (`assertItemsEqual()`).